docs/README.md

```diff
@@ -69,7 +69,8 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-ko.md)
 - Malayalam / മലയാളം
 - Nepali / नेपाली
-  - [आचार संहिता](CODE_OF_CONDUCT-np.md) 
+  - [आचार संहिता](CODE_OF_CONDUCT-np.md)
+  - [Contributing](CONTRIBUTING-np.md)
```

Comment:
This file is not there yet. Please feel free to add if you are interested

Reply:
@shrn01 but I already submit this file here #8803

Reply:
Move this change also to #8803

Reply:
@shrn01 I think it's done

Reply:
@0xYujan you can close this PR then